### DISCUSSION TITLE
How to put 2 commits in 1 commit?

### DISCUSSION THREAD

**Discussion by rishanbenahalkar** [SudhanPlayz/Discord-MusicBot#816] [Q&A]
I've seen multiple people do this and I'd like to know too, I did search online on how to do this but it was too complicating to understand, can anyone tell me how to do this?

**Comment by brianferri** [comment]
You can merge the two branches and then create a pull request for the resulting branch.

Edit: clarified the process, punctuation.

**Reply by DarrenOfficial** [reply]
or just commit it in a single branch, then make a pr on that branch

**Reply by rishanbenahalkar** [reply]
Thanks Darren, your idea worked.

**Comment by Neko-Life** [comment]
wdym "put 2 commits in 1 commit"
